lib/fastlane/plugin/wpmreleasetoolkit/helper/metadata_download_helper.rb

```diff
@@ -112,8 +117,13 @@ def handle_glotpress_download(response:, locale:, is_source:)
           UI.message("Received 301 for `#{locale}`. Following redirect...")
           download(locale, response.header['location'], is_source)
         when '429'
-          # We got rate-limited, offer to try again
-          if UI.confirm("Retry downloading `#{locale}` after receiving 429 from the API?")
+          # We got rate-limited, auto_retry or offer to try again with a prompt
+          if @auto_retry && @auto_retry_attempt_counter <= MAX_AUTO_RETRY_ATTEMPTS
+            UI.message("Received 429 for `#{locale}`. Auto retrying in #{AUTO_RETRY_SLEEP_TIME} seconds...")
+            sleep(AUTO_RETRY_SLEEP_TIME)
```

Comment:
I wonder if we wouldn't benefit from a linear or exponential wait time between attempts instead of a fixed time?

e.g. `sleep(10 + @auto_retry_attempt_counter*5)` or something?

Reply:
I've considered this one, but in my experience, it's really not necessary. I think the server just checks the number of requests in a certain amount of time, but I could be wrong.

For what it's worth, I actually wanted to go with something shorter, but since it's in CI, it doesn't really matter if it takes 1 or 2 minutes longer. Funnily enough, I tried to create the `429` error today for quite some time and I couldn't get it even once.

Happy to apply the suggestion if you feel there is enough benefit.